.github/workflows/cetlvast.yml

```diff
@@ -146,7 +146,7 @@ jobs:
       - name: Deploy to GitHub Pages
         if: ${{ github.event_name != 'pull_request' }}
         id: deployment
-        uses: actions/deploy-pages@v4.0.5
+        uses: actions/deploy-pages@v4
```

Comment:
bad bot

Reply:
Going to discard this PR and update dependabot in a separate PR.